samples/projects/quantum_backend_application/qsharp.json

```diff
@@ -0,0 +1,11 @@
+{
+    "dependencies": {
+        "ContosoBackend": {
+            "github": {
+                "owner": "microsoft",
+                "repo": "qsharp",
+                "ref": "4e7863b7e31926f3bcd5c4afeaed721d0fb0a652/samples/projects/quantum_backend_library"
```

Comment:
I know we recommend using hashes, but maybe it makes sense to just force it in with `main` as the ref here, so you don't have to update this every time you touch the backend? I can imagine that getting old...

Reply:
I reverted this to a local dependency to be compatible with the way we test samples.